static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -81,9 +81,11 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                 <% } else if(field.type === "file") { %>
                 <input name="field_<%=field.id%>"
                        type="file"
-                       accept="image/*, .pdf, .doc, .xml, .docx"
+                       accept="image/*, ,video/*, .pdf"
```

Comment:
There's a typo I think: `, ,video` should just be `, video`?

Reply:
oh yep good catch